### PR TITLE
Fix docs

### PR DESCRIPTION
**Pull request by mgeplf**
* getting a sphinx error:

    docstring of neuror.cut_plane.detection.CutPlane:1:duplicate object description of
    neuror.cut_plane.detection.CutPlane, other instance in neuror.cli, use
    :noindex: for one of them

* cleaned up imports in neuror/cut_plane/detection.py

### CODE REVIEW

**Comment by asanin-epfl** [comment]
I thought I fixed it with https://github.com/BlueBrain/NeuroR/pull/81/commits/bbca0e98ef39587d53f08b12f363372bea68350e. Did you receive the error with the latest master?

File: neuror/main.py [hunk 1]
```diff
@@ -22,8 +22,7 @@
 from nptyping import NDArray
 from scipy.spatial.distance import cdist
 
-from neuror import axon
-from neuror.cut_plane import CutPlane
```
Comment (asanin-epfl) [inline neuror/main.py:26]: So, if at some point someone brings back this import, the problem reappears. From what I understand it is rather a bug of sphinx + autosummary.

File: doc/source/api.rst [hunk 2]
```diff
@@ -15,7 +15,6 @@ API
     neuror.sanitize
     neuror.unravel
     neuror.full
-    neuror.cli
```
[the remark this answers is not in the document]
Comment (mgeplf) [reply doc/source/api.rst:18]: it's not really part of the API, IMO

File: neuror/main.py (hunk 1 above)
Comment (mgeplf) [reply neuror/main.py:26]: The doc will fail to build then, assuming that bug still exists

**Comment by mgeplf** [comment]
> I thought I fixed it with [bbca0e9](https://github.com/BlueBrain/NeuroR/commit/bbca0e98ef39587d53f08b12f363372bea68350e). Did you receive the error with the latest master?

I hadn't seen it when I started this; I'd prefer not to pin the doc version, if possible.

**Comment by asanin-epfl** [comment]
> I hadn't seen it when I started this; I'd prefer not to pin the doc version, if possible.

ok, I've removed the version constraint then. Lets see if it passes.

**Review by asanin-epfl — APPROVED** [review]

File: doc/source/api.rst (hunk 2 above)
[the remark this answers is not in the document]
Comment (asanin-epfl) [reply doc/source/api.rst:18]: `cli` actually is documented separately thanks to Andrew. I didn't notice it.

**Pull request merged**